src/main/dumpxdr.cpp

```diff
@@ -78,6 +78,43 @@ cereal_override(cereal::JSONOutputArchive& ar, const stellar::PublicKey& s,
 {
     xdr::archive(ar, stellar::KeyUtils::toStrKey<stellar::PublicKey>(s), field);
 }
+
+// A struct used to archive MuxedAccount
+// when type = KEY_TYPE_MUXED_ED25519.
+struct MuxedEd25519
+{
+    stellar::uint64 id;
+    std::string accountID;
+    template <class Archive>
+    void
+    serialize(Archive& ar)
+    {
+        ar(CEREAL_NVP(id), CEREAL_NVP(accountID));
+    }
+};
+
+void
+cereal_override(cereal::JSONOutputArchive& ar,
+                const stellar::MuxedAccount& muxedAccount, const char* field)
+{
+    MuxedEd25519 account;
```

Comment:
Nitpick: It is generally good policy to declare variables as close to where they are needed as possible (see Scott Meyers' Effective C++, Third Edition, Item 26). This should be declared in the `KEY_TYPE_MUXED_ED25519` branch, which will require you to add an explicit scope there.

That being said, I don't think you need this at all -- see the previous comment.

Reply:
Ah, that makes sense! There's no need for `account` to be visible outside the branch. I'll keep that there for now since I might delete it after all.

Reply:
Removed!